integration/spark/README.md

```diff
@@ -72,61 +72,72 @@ spark = (SparkSession.builder.master('local').appName('rdd_to_dataframe')
              .config('spark.jars.packages', 'org.postgresql:postgresql:42.2.+')
              .config('spark.extraListeners', 'io.openlineage.spark.agent.OpenLineageSparkListener')
-             .config('spark.openlineage.url', 'http://{openlineage.client.host}/api/v1/namespaces/spark_integration/')
+             .config('spark.openlineage.transport.type', 'http')
+             .config('spark.openlineage.transport.url', 'http://{openlineage.client.host}/api/v1/namespaces/spark_integration/')
              .getOrCreate())
 ```
 
 ## Arguments
 
 ### Spark Listener
 The SparkListener reads its configuration from SparkConf parameters. These can be specified on the
-command line or in the `conf/spark-defaults.conf` file. 
+command line or in the `conf/spark-defaults.conf` file.
 
-There are two ways of supplying parameters, each parameter as separate config entry or all the parameters in 
-`spark.openlineage.url`
+The following parameters can be specified in the Spark configuration:
 
-NOTE: If `spark.openlineage.url` is defined, all the parameters will be taken from it regardless of whether they are
-also defined in configuration or not.
+### General
 
-#### Config entries
+Parameters configuring the Spark integration
 
-The following parameters can be specified in the Spark configuration:
+| Parameter                                | Definition                                                                                          | Example                         |
+------------------------------------------|-----------------------------------------------------------------------------------------------------|---------------------------------
+| spark.openlineage.transport.type         | The transport type used for event emit, default type is `console`                                   | http                            |
+| spark.openlineage.namespace              | The default namespace to be applied for any jobs submitted                                          | MyNamespace                     |
+| spark.openlineage.parentJobName          | The job name to be used for the parent job facet                                                    | ParentJobName                   |
+| spark.openlineage.parentRunId            | The RunId of the parent job that initiated this Spark job                                           | xxxx-xxxx-xxxx-xxxx             |
+| spark.openlineage.appName                | Custom value overwriting Spark app name in events                                                   | AppName                         |
+| spark.openlineage.facets.disabled        | `;` separated list of facets to disable, default is `spark_unknown;` (currently must contain `;`)   | spark_unknown;spark.logicalPlan |
 
-| Parameter | Definition | Example |
-------------|------------|---------
-| spark.openlineage.host | The hostname of the OpenLineage API server where events should be reported | http://localhost:5000 |
```

Comment:
I see you removed it but still make use of this within code. If so, please consider annotating it `deprecated` instead.

Reply:
if it is still in the code then maybe it got in there during some reabase, in gereral it shouldn't be there and I'll remove it